src/components/Table/utils/columns.tsx

```diff
@@ -135,32 +135,71 @@ export function calcColumnSizes<R extends Kinded>(
     { claimedPercentages: 0, claimedPixels: 0, totalFr: 0 },
   );
 
+  function getFrUnit(w: string | number | undefined): number | undefined {
+    return typeof w === "number"
+      ? w
+      : typeof w === "undefined"
+      ? 1
+      : typeof w === "string" && w.endsWith("fr")
+      ? Number(w.replace("fr", ""))
+      : undefined;
+  }
+
+  // In the event a column defines a fractional unit (fr) as the `w` value and a `mw` value in pixels,
+  // it is possible that the min-width value will kick in and throw off our claimedPixel and totalFr calculations.
+  // Once a `tableWidth` is defined, then we can adjust the claimedPixels and totalFr based on minWidth being present for any columns
+  let adjustedClaimedPixels = claimedPixels;
+  let adjustedTotalFr = totalFr;
+  if (tableWidth) {
+    columns.forEach(({ w, mw }) => {
+      const frUnit = getFrUnit(w);
+      if (mw === undefined || frUnit === undefined) return;
+
+      const mwPx = Number(mw.replace("px", ""));
+      const calcedWidth =
+        (tableWidth - (claimedPercentages / 100) * tableWidth - adjustedClaimedPixels) * (frUnit / adjustedTotalFr);
+      // If the calculated width is less than the minWidth, then adjust the claimedPixels and totalFr accordingly
```

Comment:
It sounds like we're basically saying "you're not FR based anymore, you're actually pixel based"? If so, maybe work that into the in-source comment?